test/gradcheck.jl

```diff
@@ -235,49 +200,6 @@ end
   @test gradient(xs -> sum(inv, [x^2 for x in xs]), ones(2)) == ([-2, -2],)
 end
 
-@testset "conv: spatial_rank=$spatial_rank" for spatial_rank in (1, 2, 3)
```

Comment:
It might also be worth keeping NNlib as a test dependency, so that these tests can stay. Possibly moved around. But maybe they catch edge cases which would be easy to miss in re-writing.

Reply:
They are still in NNlib, so will be caught sooner or later. We could run NNlib tests as an integration test as part of CI, but I am not sure it's worth it, since these don't really test anything that isn't already tested another way.

Reply:
Oh I'm sorry I didn't scroll down far enough, they are indeed copied by your PR!